microservice/pom.xml

```diff
@@ -15,7 +15,7 @@
         <changelist>-SNAPSHOT</changelist>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>1.8</java.version>
+        <java.version>9</java.version>
```

Comment:
this change was unintended I guess.

Reply:
Looking into it. I barely remember I had to change the java version at a point in time, but this might indeed be obsolete.

Reply:
Was obsolete. Works.